tests/lib/rules/strict-component-boundaries.js

```diff
@@ -25,6 +25,14 @@ ruleTester.run('strict-component-boundaries', rule, {
       code: `import {someThing} from '../OtherComponent';`,
       parserOptions,
     },
+    {
+      code: `import fs from 'fs';`,
+      parserOptions,
+    },
+    {
+      code: `import eslint from 'eslint';`,
+      parserOptions,
+    },
```

Comment:
Would be good to test that this actually works for a path that is in node modules and that does not adhere to our rules (something/MyComponent)

Reply:
see https://github.com/Shopify/eslint-plugin-shopify/pull/140#discussion_r207691748

Reply:
I need to actually install `something/MyComponent`, otherwise I don't think this will work.

Reply:
See example here: 
https://github.com/Shopify/eslint-plugin-shopify/blob/master/tests/lib/rules/jsx-no-hardcoded-content.js#L156

Reply:
🙇